### PR TITLE
TFSID 1482018: [SDK] Update Manifest XML to allow multiple oauth-config files for Multiple IDP

### PR DESCRIPTION
**Pull request by ethan-pyke**
**Things To Remember:** 

1. Be sure to merge into the right branch. The choices are:
* **`master`** if your PR runs on the current version of Tableau.
* **`dev-2021.2`**  
2. You should update at least one of the following:
* The changelog of the project you modified.
* The root `README.md` if the PR is a release of the SDK, Connector Packager, or TDVT.  
=====
Update manifest XSD to allow more than one `oauth-config` fields for Multiple IDP and to set the minimum Tableau version to 2021.4 if present.

### CODE REVIEW

**Review by pvanderknyff — APPROVED** [review]
LGTM from a packaging perspective

File: connector-packager/tests/test_jar_packager.py [hunk 1]
```diff
@@ -277,3 +277,51 @@ def test_jdk_create_jar_oauth_with_configid(self):
...
+        path_to_extracted_oauth_config = dest_dir / "oauth-config.xml"
+        self.assertTrue(os.path.isfile(path_to_extracted_oauth_config), "extracted oauth-config.xml file doesn't exist")
+
+        # test to see if oauth-config-copy.xml file was packaged
+        args = ["jar", "xf", package_name, "oauth-config-copy.xml"]
+        p = subprocess.Popen(args, cwd=os.path.abspath(dest_dir))
+        self.assertEqual(p.wait(), 0, "can not extract oauth-config-copy.xml from taco")
+        path_to_extracted_oauth_config = dest_dir / "oauth-config-copy.xml"
+        self.assertTrue(os.path.isfile(path_to_extracted_oauth_config), "extracted oauth-config-copy.xml file doesn't exist")
+
```
Comment (rosswbrown) [inline connector-packager/tests/test_jar_packager.py:325]: Should there be any assertion around `oauth-config-with-id.xml`?

Comment (ethan-pyke) [reply connector-packager/tests/test_jar_packager.py:325]: Do you mean like should I check for the config file within the `test_jdk_create_jar_oauth_with_configid` method?

Comment (rosswbrown) [reply connector-packager/tests/test_jar_packager.py:325]: I was just trying to figure out if it was actually needed or not by the test. If it is exercising a unique use case you can drop this comment.

**Review by rosswbrown — APPROVED** [review]

**Pull request merged**